01-intro.md

```diff
@@ -131,3 +129,9 @@ all share the same fundamental concepts as Make.
 > with text to produce reports and papers for publication.
 >
 > Make's fundamental concepts are common across build tools.
+
+[GNU Make](http://www.gnu.org/software/make/) is a free, fast, 
+well-documented, very popular and highly extended one of some 
+Make impliments. From now on, we will focus on it, then when 
```

Comment:
Hi Michael,


Thanks for pointing me the typo!


I have re-done everything, deleted the old pull request, and put a new pull request as #30. Please have a look.

Best Wishes!

Gang Liu

http://www.hpcvl.org
Queen's University
Kingston, ON
613-533-6000 ext.78387
gang.liu@queensu.ca

Dynamical Equations of Period Vectors:
Can. J. Phys. Vol.93 No.9: 974-978 (2015)
http://dx.doi.org/10.1139/cjp-2014-0518
http://arxiv.org/abs/cond-mat/0209372v16
https://github.com/LiuGangKingston






________________________________
From: Michael Jackson <notifications@github.com>
Sent: 16 November 2015 06:05
To: swcarpentry/make-novice
Cc: Gang Liu
Subject: Re: [make-novice] Comments for GNU Make updated a little bit. (#26)


In 01-intro.md<https://github.com/swcarpentry/make-novice/pull/26#discussion_r44911107>:

[https://avatars0.githubusercontent.com/u/12719634?v=3&s=400]<https://github.com/swcarpentry/make-novice/pull/26#discussion_r44911107>

Comments for GNU Make updated a little bit. by LiuGangKingston · Pull Request #26 · swcarpentry/make-novice
In order to be a qualified instructor of software carpentry.
Read more...<https://github.com/swcarpentry/make-novice/pull/26#discussion_r44911107>



> @@ -131,3 +129,9 @@ all share the same fundamental concepts as Make.
>  > with text to produce reports and papers for publication.
>  >
>  > Make's fundamental concepts are common across build tools.
> +
> +[GNU Make](http://www.gnu.org/software/make/) is a free, fast,
> +well-documented, very popular and highly extended one of some
> +Make impliments. From now on, we will focus on it, then when


Please change "impliments" to "implements" then I'll merge.

-
Reply to this email directly or view it on GitHub<https://github.com/swcarpentry/make-novice/pull/26/files#r44911107>.